lmod_proxy/config.py

```diff
@@ -22,7 +22,15 @@
 
     # Base URL for which the gradebook API lives and accepts
     # certificate authentication
-    'LMODP_URLBASE': 'https://learning-modules.mit.edu:8443/'
+    'LMODP_URLBASE': 'https://learning-modules.mit.edu:8443/',
+
+    # Direct path to apache htpasswd file to use for basic auth
```

Comment:
no, we are using htpasswd formatted password files from apache to enable authentication, i.e. the output of:
`htpasswd -c .htpasswd cgee` that you would do to protect a folder in apache.  It is as close to a standard as we have, as nginx and several other Web servers use the same format.